proxies/java/src/main/java/com/dvc_harness/proxy/controller/ProxyController.java

```diff
@@ -28,6 +30,29 @@ public Spec spec() {
         return new Spec();
     }
 
+    @PostMapping("/user")
+    public BaseResponse user(@RequestBody UserRequestBody body, HttpServletResponse response) {
+        try {
+            User sdkUser = User.builder()
+                .userId(body.user_id)
+                .name(body.name)
+                .customData(body.customData)
```

Comment:
i think you might as well just list every property here

Reply:
but its so ugly 😭 okay tho

Reply:
thats Java babyyyy

Reply:
you could probably write some fancy reflection thing to pull the correct method name based on the name of each property in the request

Reply:
id rather just get it done lol